kubernetes/gke-utility/helm/ingress-nginx.yaml

```diff
@@ -1,4 +1,5 @@
 controller:
+  allowSnippetAnnotations: true # Required for OAuth2 Proxy
```

Comment:
I'm working on exposing ArgoCD via OAuth2 Proxy(using GitHub OIDC Login) and it requires custom lua proxy code (L00l)

https://oauth2-proxy.github.io/oauth2-proxy/configuration/integration/#forwardauth-with-static-upstreams-configuration